src/vm/interoputil.cpp

```diff
@@ -5161,14 +5161,6 @@ ClassFactoryBase *GetComClassFactory(MethodTable* pClassMT)
 
     if (pClsFac == NULL)
     {
-        //
-        // Collectible types do not support com interop
-        //
-        if (pClassMT->Collectible())
-        {
-            COMPlusThrow(kNotSupportedException, W("NotSupported_CollectibleCOM"));
```

Comment:
Delete the `NotSupported_CollectibleCOM` message?

Reply:
Thanks for spotting that.